.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+	"extends": "wikimedia",
+	"env": {
+		"browser": true,
+		"jquery": true
+	},
+	"globals": {
+		"dataTypes": false,
+		"define": false,
+		"QUnit": false
```

Comment:
I believe it is better to have test-specifc rules, environments, globals etc defined in test-only eslintrc file. I'll add a commit to the PR to show what I mean.